src/@carbon-info/views/Data/Data.tsx

```diff
@@ -1,217 +1,243 @@
 import { FadeAndSlide } from "@carbon-info/components";
 import { useContentful } from "@carbon-info/hooks";
 import { isWidth } from "@carbon-info/utils/environment";
-import { StyleUtils } from "@carbon-info/utils/styles";
-import { Divider, Grid, makeStyles, Paper, Theme, Typography } from "@material-ui/core";
+import {
+	Box,
+	Divider,
+	Grid,
+	makeStyles,
+	Paper,
+	Theme,
+	Typography,
+} from "@material-ui/core";
 import clsx from "clsx";
 import React, { useEffect } from "react";
 import { useInView } from "react-intersection-observer";
 import { AnimateKeyframes } from "react-simple-animate";
 import { RollingNum } from "./component";
-import cosmosSDK from "@carbon-info/assets/icons/cosmosSDK.svg";
+import Banner from "@carbon-info/components/Banner";
+import { DemexLogo } from "@carbon-info/assets";
 
 export interface DataInfo {
-  value: string,
-  description: string,
-  toCountUp: boolean,
+	value: string;
+	description: string;
+	toCountUp: boolean;
 }
 
 const Data: React.FC = () => {
-  const classes = useStyles();
-  const { ref, inView } = useInView({
-    /* Optional options */
-    threshold: 0.4,
-    triggerOnce: true,
-  });
+	const classes = useStyles();
+	const { ref, inView } = useInView({
+		/* Optional options */
+		threshold: 0.4,
+		triggerOnce: true,
+	});
 
-  const widthXs = isWidth("xs");
+	const widthXs = isWidth("xs");
 
-  const { data } = useContentful({
-    contentType: "carbonWebsiteInfo",
-  });
+	const { data } = useContentful({
+		contentType: "carbonWebsiteInfo",
+	});
 
-  const [tableInfo, setTableInfo] = React.useState<DataInfo[]>([]);
-  useEffect(() => {
-    const results: DataInfo[] = [];
-    if (!data && !inView) return;
-    async function fetchDataItems() {
-      const content = await data as any;
+	const [tableInfo, setTableInfo] = React.useState<DataInfo[]>([]);
+	useEffect(() => {
+		const results: DataInfo[] = [];
+		if (!data && !inView) return;
+		async function fetchDataItems() {
+			const content = (await data) as any;
 
-      if (content && Array.isArray(content.items)) {
-        content.items.forEach((o: any) => {
-          let data: DataInfo = {
-            value: o.fields.value,
-            description: o.fields.key,
-            toCountUp: o.fields.key === "On-Chain_Transactions" || o.fields.key === "Total_Value_Locked" || o.fields.key === "Total_SWTH_Staked",
-          };
-          results.push(data);
-        });
-      }
-      setTableInfo(results);
-    }
-    fetchDataItems();
-  }, [data]);
+			if (content && Array.isArray(content.items)) {
+				content.items.forEach((o: any) => {
+					let data: DataInfo = {
+						value: o.fields.value,
+						description: o.fields.key,
+						toCountUp:
+							o.fields.key === "On-Chain_Transactions" ||
+							o.fields.key === "Total_Value_Locked" ||
+							o.fields.key === "Total_SWTH_Staked",
```

Comment:
This can be shortened to
```typescript
toCountUp: [
  "On-Chain_Transactions",
  "Total_Value_Locked",
  "Total_SWTH_Staked"
].includes(o.fields.key),
```